contrib/icecream/files/tmpfiles.conf

```diff
@@ -0,0 +1,3 @@
+# icecream tmpfiles setup
+
+d /var/log/icecc 1770 _icecc _icecc -
```

Comment:
upstream uses 0755 here as far as i can tell, which is i think fine? there isn't anything sensitive in the logs